docs/UserGuide.md

```diff
@@ -127,6 +127,13 @@ Format: `addTxn INDEX amt/AMOUNT desc/DESCRIPTION [date/DATE] [status/STATUS] [c
 * The `STATUS` accepts case-sensitive string that is either 'Done' or 'Not Done'. 
 * The `CATEGORY` accepts non-empty strings that are alphanumeric with spaces. Category will be capitalised automatically.
 
+> [!IMPORTANT]
+> Two identical Transactions with duplicated fields cannot be added into the transaction book.
```

Comment:
I recommend to put this in the bulb and tip, can consider rephrasing to:

For consistency and to avoid redundancy, identical transactions with the same details across all fields will not be added to the transaction book. This ensures that each entry remains unique, preventing accidental duplicates and maintaining the clarity of transaction records. If a similar transaction occurs on a different occasion in the same day, we recommend users to tweak the desc field to reflect the specific context. For example:

`addTxn 1 amnt/2.50 desc/sean owes me for morning latte`
`addTxn 1 amnt/2.50 desc/sean owes me for afternoon latte`

Reply:
Related issue: https://github.com/AY2425S1-CS2103-F13-1/tp/issues/199

Reply:
Are you okay with using
:exclamation: **Important:**
instead just to emphasise this fact as this seems like a pretty important note for users to consider